app/Commands/BaseCommand.php

```diff
@@ -53,6 +54,19 @@ public function handle(): int
             }
 
             return $this->action();
+        } catch (ValidationException $e) {
```

Comment:
It can be confusing user experience here as the validation field names don't match the labels the user completes. 
```
Site User []:
 >

 Db Name []:
 >

 Db User []:
 >
```
vs
``` 
site_user                The site user field is required.
database.name            The database name field is required.
database.username        The database username field is required.
```

Can we do something to get them matching? My first thought is to have a translation array we run them through.

Reply:
I'm not sure I understand the nature of the confusion. Would changing the "Db Name" and "Db User" labels to "Database Name" and "Database Username" resolve the confusion?

Reply:
✅